src/test/resources/org/wildfly/test/integration/vdx/transformations/messaging/InvalidAddressSettingFullPolicy.groovy

```diff
@@ -0,0 +1,15 @@
+attr = [name: '#']
+attr['dead-letter-address'] = 'jms.queue.DLQ'
+attr['expiry-address'] = 'jms.queue.ExpiryQueue'
+attr['max-size-bytes'] = '10485760'
+attr['page-size-bytes'] = '2097152'
+attr['page-size-bytes'] = '2097152'
```

Comment:
attr['page-size-bytes'] = '2097152' is duplicated

it's not clear what is invalid in the config ... only people with messaging context would know ... please add comment